torch/fx/proxy.py

```diff
@@ -164,19 +164,15 @@ def create_proxy(self, kind: str, target: Target, args: Tuple[Any, ...], kwargs:
         if fx_traceback.has_preserved_node_meta():
             current_meta: Dict[str, Any] = fx_traceback.get_current_meta()
 
-            # Explicitly set the stack_trace, nn_module_stack and source_fn on the node.meta
-            # If other meta fields are needed, they can be added here
             stack_trace = current_meta.get("stack_trace")
```

Comment:
nit, stack_trace can also be included in the copy_meta_fields

Reply:
Stack trace is stored as an attribute directly on node, while the other metadata are stored in the meta field.